### PR TITLE
[MM-32692]: Feature: Add two more CRT options in System Console: default_on & always_on

### PR DESCRIPTION
**Pull request by vish9812**
#### Summary
Feature: Add two more CRT options in System Console: default_on & always_on

#### Ticket Link
JIRA: https://mattermost.atlassian.net/browse/MM-32692

#### Related Pull Requests
- Has server changes https://github.com/mattermost/mattermost-server/pull/20088

#### Release Note
```release-note
* Add Default on and Always on options for CRT in system console
```

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -816,8 +816,10 @@
...
   "admin.experimental.clientSideCertEnable.title": "Enable Client-Side Certification:",
+  "admin.experimental.collapsedThreads.always_on": "Always On",
   "admin.experimental.collapsedThreads.default_off": "Enabled (Default Off)",
-  "admin.experimental.collapsedThreads.desc": "When enabled (default off), users must enable collapsed reply threads in Settings. When disabled, users cannot access Collapsed Reply Threads. Please review our [documentation for known issues](!https://docs.mattermost.com/messaging/organizing-conversations.html) and help provide feedback in our [community channel](!https://community-daily.mattermost.com/core/channels/folded-reply-threads).",
+  "admin.experimental.collapsedThreads.default_on": "Enabled (Default On)",
+  "admin.experimental.collapsedThreads.desc": "When enabled (default off), users have the option enable Collapsed Reply Threads in Account Settings. When enabled (default on), users see Collapsed Reply Threads by default and have the option to disable it in account settings. When always on, users are required to use Collapsed Reply Threads and cannot disable it.",
```
Comment (esethna) [inline i18n/en.json:822]: Please capitalize the second: "Account Settings"

**Comment by esethna** [comment]
Assuming this needs a server PR as well, but please ensure that the user account setting for CRT is hidden if "Always On' is selected

**Review by esethna — APPROVED** [review]

**Review by koox00 — APPROVED** [review]

File: i18n/en.json (hunk 1 above)
Comment (vish9812) [reply i18n/en.json]: Done.

File: packages/types/src/config.ts [hunk 2]
```diff
@@ -736,7 +736,7 @@ export type PluginSettings = {
     Directory: string;
     ClientDirectory: string;
     Plugins: Record<string, any>;
-    PluginStates: Record<string, {Enable: boolean}>;
+    PluginStates: Record<string, { Enable: boolean }>;
```
Comment (AshishDhama) [inline packages/types/src/config.ts:739]: nit: try to avoid unnecessary changes

**Review by AshishDhama — APPROVED** [review]
LGTM !!!

**Review by jgilliam17 — COMMENTED** [review]
Thanks @vish9812 
Tested, looks good.
I have one suggestion for the help text and I also added a test case [MM-T4798](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin:com.kanoah.test-manager__main-project-page#!/testCase/MM-T4798) that would be good to automate.

**Comment by jgilliam17** [comment]
@vish9812 Can you please resolve the conflicts so we can merge.
I'll create a follow up ticket for implementing the E2Es as not to hold this PR up.

**Comment by jgilliam17** [comment]
Sorry, I should have refreshed the browser first 🙂, didn't see you updated already.

**Review by jgilliam17 — APPROVED** [review]
Follow up ticket for E2Es https://mattermost.atlassian.net/browse/MM-44096

**Comment by jgilliam17** [comment]
@vish9812 I missed that E2E run triggered earlier, never completed successfully. Removing my approval pending E2Es.

**Comment by saturninoabril** [comment]
@jgilliam17 @vish9812 There was a problem with Cypress E2E but it's all good now. You'll find the result at https://automation-dashboard.vercel.app/cycles/24cb32d3-f81c-4ec1-8144-45001e2b7c7c and in the community channel.

**Review by jgilliam17 — APPROVED** [review]

**Comment by jgilliam17** [comment]
@vish9812 I didn't merge due to `do not merge` label. Please merge both webapp and server PR when ready - thanks 🙂
